docs/VDTS_Setup.md

```diff
@@ -0,0 +1,25 @@
+# Setting up Virtual Device
+## Background
+We need a Virtual Device in order to test your Alexa skills. What is a Virtual Device (VD)? it is like an Amazon Echo but made with code.
```

Comment:
Don't use the shorthand VD - VD is short for venereal disease en ingles - very bad connotation

Reply:
"It is like a physical Amazon Echo, but one that can be interacted with programmatically"